rom/dev/src/pcr.rs

```diff
@@ -129,6 +130,7 @@ pub(crate) fn extend_pcrs(
 ///
 #[cfg_attr(not(feature = "no-cfi"), cfi_mod_fn)]
 pub fn log_pcr(
+    pcr_log: &mut [PcrLogEntry; 17],
```

Comment:
Suggest using a const instead of hardcoding.

Reply:
Replaced with a PcrLogArray typedef.